.build/parent-pom-template.xml

```diff
@@ -774,6 +774,13 @@
       </dependency>
       <!-- end of chronicle-queue -->
 
+      <dependency>
+        <groupId>software.amazon.cryptools</groupId>
+        <artifactId>AmazonCorrettoCryptoProvider</artifactId>
+        <version>[2.0, 3.0)</version>
```

Comment:
This is the most probably wrong, we can not use range versions. We need repeatable builds so we have to make it version specific.